frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/ol-map-manager.ts

```diff
@@ -310,5 +316,22 @@ export class OlMapManager {
             materialFeatureManager,
             viewportFeatureManager,
         ];
+
+        this.featureNameFeatureManagerDictionary.set(
+            'vehicle',
+            vehicleFeatureManager
+        );
+        this.featureNameFeatureManagerDictionary.set(
+            'patient',
+            patientFeatureManager
+        );
+        this.featureNameFeatureManagerDictionary.set(
+            'personnel',
+            personnelFeatureManager
+        );
+        this.featureNameFeatureManagerDictionary.set(
+            'material',
+            materialFeatureManager
+        );
```

Comment:
Could you please add all feature managers to this map or explicitly state (in a comment above these lines, a JSDoc comment for the variable or a more descriptive variable name) when a feature manager has to be added and when not.

I would prefer adding all feature managers since it would make it easier to add more features to the `changedLayers` of a popup.

Reply:
Done